src/commands/list.rs

```diff
@@ -78,6 +79,15 @@ pub fn list_archive_contents(
 
             Box::new(crate::archive::zip::list_archive(zip_archive))
         }
+        SevenZip => {
+            let a = RefCell::new(Vec::new());
```

Comment:
If https://github.com/dyz1990/sevenz-rust/pull/17 gets merged, you can remove the `RefCell` wrap.

Reply:
fixed